twitchpopups.htm

```diff
@@ -1,6 +1,35 @@
 <!DOCTYPE HTML>
 <html>
 <head>
+
+    <script type="text/javascript">
+
+        // Change Limmy to your Twitch channel

```

Comment:
get rid of these comments. code should be clean and readable without the aid of comments

Reply:
For a tool like this I think the comments are perfectly fine.

Reply:
It's nonsense even as a general statement, but especially in this case, considering target audience, it's just obviously untrue.

Reply:
> get rid of these comments. code should be clean and readable without the aid of comments

This isn't code, it's configuration.